src/DynamoCoreWpf/DynamoGeometryModel3D.cs

```diff
@@ -0,0 +1,290 @@
+﻿using System;
```

Comment:
This type is brought over from the Helix code-base. It's the underlying class for every mesh drawn in Dynamo.